src/field_theory/finite.lean

```diff
@@ -159,3 +159,20 @@ let ⟨a, b, hab⟩ := zmodp.sum_two_squares (show nat.prime n,
 end⟩
 
 end char_p
+
+open_locale nat
+open zmod
+
+@[simp] lemma zmod.pow_totient {n : ℕ+} (x : units (zmod n)) : x ^ φ n = 1 :=
```

Comment:
I think that both these lemmas could have a docstring that mention Euler (and maybe Fermat):
https://en.wikipedia.org/wiki/Euler%27s_theorem